README.rst

```diff
@@ -73,7 +73,7 @@ If you want to contribute, install your cloned fork:
 
     git clone git@github.com:<your_fork>/openwisp-utils.git
```

Comment:
I think *<your_username>* will be better, than *<your_fork>* but it is nit-picking :P

Reply:
Nitpicking is also necessary.